reports/report_builder.py

```diff
@@ -149,6 +149,8 @@ def __init__(self, form):
             self.report_type = 'global'
 
         self.country_list = [code for code, name in self.countries]
+        all_regions = add_transnational_to_regions(self.regions)
+        self.all_region_list = [name for id, name in all_regions]
```

Comment:
We shouldn't have this on `self`, just like we don't have `all_regions` on `self`... it's query dependent.